Essentials/src/com/earth2me/essentials/commands/Commandrecipe.java

```diff
@@ -26,18 +25,16 @@
     public Commandrecipe() {
         super("recipe");
     }
-    
-    private void disableCommandForVersion1_12() throws Exception {
-        VersionUtil.BukkitVersion version = VersionUtil.getServerBukkitVersion();
-        if (version.isHigherThanOrEqualTo(VersionUtil.v1_12_0_R01)
-            && !ess.getSettings().isForceEnableRecipe()) {
-            throw new Exception("Please use the recipe book in your inventory.");
-        }
-    }
 
     @Override
     public void run(final Server server, final CommandSource sender, final String commandLabel, final String[] args) throws Exception {
-        disableCommandForVersion1_12();
+        try {
+            Class.forName("com.destroystokyo.paper.event.player.PlayerRecipeBookClickEvent");
+        } catch (ClassNotFoundException e) {
+            sender.sendMessage(tl("unsupportedFeature"));
```

Comment:
This breaks the command on 1.11 and lower. Maybe add a 1.12 check here?

Reply:
Class::forName doesn't work? I'll look into this.

Reply:
@pop4959 No, this command still works (as of the latest build) on 1.11.x and below, but by removing the 1.12 version check entirely we drop support for this command on old versions of MC. These versions are the ones that don't have the recipe book, and so they're the versions where this command is the *most* useful.

Reply:
Ah right, completely overlooked that. I'll re-add this check ASAP.